docs/reference-guides/project_onboarding_procedure.md

```diff
@@ -0,0 +1,74 @@
+# Project Onboarding Procedure
+
+Below are step-by-step instructions on how to onboard a project to report test metrics.
+
+## Prerequisites
+
+To report test metrics for a project, ensure the following requirements are met:
+
+- The project uses [CircleCI][CircleCI] for test execution, and members of the ETE team have access to the
+  CircleCI pipelines.
+- Testing and coverage results are stored as artifacts in CircleCI jobs:
+  - Test results must be in JUnit format.
+  - Coverage results must be in JSON format.
+  - Supported test frameworks are listed in the 
+    [Metric Interpretation Guide][Metric Interpretation GuideI].
+
+## 1. Add CircleCI Pipelines to the Config File and Scrape Historical Data
+
+- Add the new CircleCI pipelines under the `circleci_scraper` section in your local `config.ini` and
+  `config.ini.sample`. Push changes to the `config.ini.sample` to the repository to keep
+  contributors up-to-date.
+- Execute the `circleci_scraper` using the following command. Ensure the `days_of_data` option in
+  the `circleci_scraper` section of `config.ini` is omitted to scrape all available historical data.
+  Test and coverage data will be stored locally in the `test_result_dir` (typically called
+  `raw_data` at the project root).
+  ```shell
+  make run_circleci_scraper
+  ```
+- Once scraping is complete, compress the contents of `test_result_dir`, and replace the
+  corresponding file in the [ETE team folder][ETE Drive].
```

Comment:
Do we still need to do this now that we're updating BigQuery?

Reply:
Yes. This will need to continue until Phase 3 is complete.